database/entity/0070-add_dlt_transactions_table/Transaction.ts

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-use-before-define */
+import { Decimal } from 'decimal.js-light'
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, OneToOne, JoinColumn } from 'typeorm'
+import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
+import { Contribution } from '../Contribution'
+import { DltTransaction } from './DltTransaction'
```

Comment:
This file should not be modified according to the migration.

Reply:
I do not understand this argument.
the only change here is to add an optional OneToOne relation to the dlt_transactions table. So i gues the current changes will have no effect.
Which migration do you mean?